libraries/WiFi/src/WiFiGeneric.h

```diff
@@ -27,10 +27,25 @@
 #include <esp_event_loop.h>
 #include <functional>
 #include "WiFiType.h"
+#include "IPAddress.h"
+#include <wifi_provisioning/manager.h>
```

Comment:
what library provides this header?

Reply:
https://github.com/espressif/arduino-esp32/blob/master/tools/sdk/include/wifi_provisioning/wifi_provisioning/manager.h

Reply:
https://github.com/espressif/arduino-esp32/blob/master/tools/sdk/include/wifi_provisioning/wifi_provisioning/manager.h